qdraw/starsky
# Removal of Directories

# PR Details 
## Description / Motivation and Context

To test: 

- todo: test removal directories (server mode)
- test use system remove directory
- undo remove of directory (and should re-create folder)



## Related Issue






## How Has This Been Tested





- [ ] C# Unit tests 
- [ ] Typescript Unit tests 
- [ ] Other Unit tests
- [ ] Automatic End2end tests (starsky-tools/end2end)


## Types of changes



- [ ] Added _for new features_
- [ ] Breaking change _fix or feature that would cause existing functionality to change_
- [ ] Changed _for non-breaking changes in existing functionality for example docs change / refactoring / dependency upgrades_
- [ ] Deprecated _for soon-to-be removed features_
- [ ] Removed _for now removed features_
- [ ] Fixed _for any bug fixes_
- [ ] Security _in case of vulnerabilities_


## Checklist




- [ ] I have updated the documentation accordingly (update when needed)
- [ ] I have updated the **./history.md** document
